1-foundations/2-control-flows-and-functions/foundations-s2.ipynb

```diff
@@ -726,18 +1268,264 @@
     "print(\"{} million people live in {}. Data as of {}\".format(query_population(\"BRA\", \"SP.POP.TOTL\", 2019), 'BRA', 2019))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "41fb3ec1-5330-49c4-9675-23a6d7a5effa",
+   "metadata": {},
+   "source": [
+    "### 2.5.3 Local and Global variables (a short detour)"
```

Comment:
I agree that it is important that we mention that a variable created in a function cannot be accessed outside of it. But maybe it is better to just talk about scope and not get into `global`/`local` and be careful to say anything that seems to recommend these beginners to start declaring global variables.

Reply:
Fair point!